### DISCUSSION TITLE
Combined plugins: RenderFile & HTML <base> (v2.0.0-canary.15)

### DISCUSSION THREAD

**Discussion by TigersWay** [11ty/eleventy#2548] [Q&A] [answered]
Hi,
~~Can someone confirm that using `renderFile` shorcode while `EleventyHtmlBasePlugin` is loaded, is working?
Preparing an update, `EleventyHtmlBasePlugin` seems to be working great but not inside rendered file(s).
Thanks~~

Sorry! I manage to solve this.

**Comment by pdehaan** [comment, marked as answer]
Worked for me https://github.com/pdehaan/11ty-2548

**Reply by TigersWay** [reply]
@pdehaan You're right it's working. Trouble was with Chrome and its autocomplete feature.
